src/components/StickySectionHeader/index.js

```diff
@@ -30,6 +30,7 @@ const useStyles = makeStyles(theme => ({
   }),
   label: {
     padding: `${theme.spacing(0.5)}px ${theme.spacing(2)}px`,
+    fontWeight: 400,
```

Comment:
Is there a specific reason for this? I was trying to keep the sticky headers consistent with the ones on the event list and team list pages.

Reply:
I think it looks better. What do you think?